pypesto/visualize/sampling.py

```diff
@@ -91,6 +92,215 @@ def sampling_fval_trace(
     return ax
 
 
+def sampling_prediction_profiles(result: Result,
```

Comment:
the colors are hard to distinguish ... not sure if that can be improved ...

Reply:
for this example the uncertainties are quite low... the different cmaps I tried were all hard to distinguish
but if you think that adding a kwargs with different color map names would help, I could implement this